python/rascaline/rascaline/utils/_helpers.py

```diff
@@ -0,0 +1,8 @@
+from rascaline.utils import _dispatch
```

Comment:
@Luthaf I am not super happy with the name. You have a better idea how we can organize the `_dispatch`

Reply:
with the `_dispatch` name?

Reply:
Also, this is different from what we do in equistore.torch.operations: over there the `_classes` module is only used to define the classes, `_dispatch` is imported as usual. 

The trick there is to start by creating a custom `equistore.torch.operations._classes` module, and registering it in `sys.module`. Then when importing the rest of the code, `from ._classes import XXX`  resolves to this custom module without re-importing the file.